### PR TITLE
Ingawei/undo delete wrapped

### PR DESCRIPTION
**Pull request by ingawei**
Added wrapped back
image
Would appreciate a sanity check to see if data I pulled is actually in line with reality lol, please see comments

### CODE REVIEW

File: backend/api/src/get-max-min-profit-2024.ts
Comment (ingawei) [inline backend/api/src/get-max-min-profit-2024.ts:1]: This one is supposed to get the contract where you made the most profit, and the contract where you made the least profit in 2024. Filtered by contracts that resolved in 2024

**Review by sipec — APPROVED** [review]
looks sane to me

File: backend/api/src/get-max-min-profit-2024.ts
Comment (IanPhilips) [reply backend/api/src/get-max-min-profit-2024.ts:1]: Looks good! I don't think you need to selecet has-yes-shares and no shares

**Review by IanPhilips — APPROVED** [review]
Looks good! Excited to see mine

**Pull request merged**